src/services/inplayer.checkout.service.ts

```diff
@@ -164,6 +164,7 @@ export const directPostCardPayment = async (cardPaymentPayload: CardPaymentData,
 
   try {
     if (isSVODOffer(order)) {
+      // @ts-ignore
```

Comment:
Update the request params type of `InPlayer.Subscription.createSubscription` by changing `number: number;` to `number: string;`. instead of using `ts-ignore`.

Reply:
that's the plan when we make sure this is the cause for the tests to fail.